packages/sanity/src/core/preview/utils/getPreviewValueWithFallback.tsx

```diff
@@ -47,12 +47,12 @@ export const getPreviewValueWithFallback = ({
     }
   } else {
     switch (true) {
-      case typeof resolveBundlePerspective(perspective) !== 'undefined' || isVersionId(value._id):
-        snapshot = version || draft || published
-        break
       case perspective === 'published':
         snapshot = published || draft
         break
+      case typeof perspective !== 'undefined' || isVersionId(value._id):
```

Comment:
don't we need here a case to handle the drafts? Or maybe this should be
```suggestion
      case typeof perspective !== 'undefined' && perspective !== "drafts" || isVersionId(value._id):
```

Reply:
Even though the client supports it, it should never be `drafts` in the studio code as that's that's always the default